multigen/pipes.py

```diff
@@ -120,12 +125,14 @@ def setup(self, steps=50, **args):
 
 class Prompt2ImPipe(BasePipe):
 
-    def __init__(self, model_id, lpw=False, **args):
+    def __init__(self, model_id: str,
+                 pipe: Optional[StableDiffusionPipeline] = None,
```

Comment:
I would remove `pipe` from the child classes. While it has downsides, explicitly listing all the parameters in child classes is a burden. In this PR, only two child classes have `pipe` parameter, but not others. We either need to add `pipe` to all child classes, or remove in all of them.